test/Microsoft.AspNetCore.TestHost.Tests/TestServerTests.cs

```diff
@@ -14,6 +14,7 @@
 using Microsoft.AspNetCore.Http.Features;
```

Comment:
If overriding services for testing is one of the main scenarios, there should be a test for that.